docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+
+## Actor default capabilities
+
+Similar to primitives, actors always have a `tag` default capability. Like primitives, it is not possible to specify a different default capability for tags.
+
+```pony
+actor Actor
+```
+
+has `tag` capabilities
+
+```pony
+actor ref Actor
+```
+
+is illegal
+
+## Class default capabilities
+
+By default, a class has `ref` capability. Therefore, these two class definitions are equivalent:
+
+```pony
+class ref RefClass
+    ...
+
+class RefClass
+    ...
+```
+
+But whether specified by default or explicitly as `class ref` (or `class val` or any other capability) what does it actually mean? A class or actor's default capability only applies when you _define_ a field or local type. This can happen either as `let` or `var`, or when specifying a parameter on a function or behaviour.
+
+## Field, Variable, and Parameter capabilities
+
+For example, `String` is defined in the standard library as `class val String`. So it's default capability is the immutable shareable `val` capability. That means that all the `String`s in the following actor definition could be replaced with `String val`:
+
+```pony
+actor UsesStrings
+    let s1: String
+    var s2: String
+
+    new create(s1': String, s2': String) =>
+        s1 = s1'
+        s2 = s2'
+
+    be do_thing(s3: String) =>
+        None
+
+```
+
+However `StringBytes` is defined as `class ref`, so all the `StringBytes`' in the following class definition are equivalent to `StringBytes ref`:
+
+```pony
+class UsesStringBytes
+    let s1: StringBytes
+    var s2: StringBytes
+
+    new create(s1': StringBytes, s2': StringBytes) =>
+        s1 = s1'
+        s2 = s2'
+
+    fun do_thing(s3: StringBytes) =>
+        None
+```
+
+Notably, however, the default capability on a class **does not** influence the constructors for that class.
+
+## Constructor capabilities
+
+Let's start with the easy defaults first: A primitive constructor _always_ returns an object with `val` capabilities, and an actor constructor _always_ returns an object with `tag` capabilities. In fact, it is forbidden to put a capability after `new` when defining a primitive or actor.
+
+Classes are a little more flexible. The capability returned by a `new` constructor default to a `ref` if you are constructing a `class`. This is true regardless of what the default capability for the class is, but you have the option to specify a different capability by explicitly typing the constructor as e.g `new var create` or `new iso create_iso`.
+
+Consider the following class:
+
+```pony
+class val MyClass
+    let foo: String
+
+    new create_ref(foo': String) =>
+        foo = foo'
+
+    new val create_val(foo': String) =>
+        foo = foo'
+```
+
+This class has two constructors. The first one, `create_ref` returns a `MyClass` with `ref` capabilities because `ref` is always default for class constructors. The second one `create_val` explicitly returns a `MyClass` with `val` capabilities.
```

Comment:
```suggestion
This class has two constructors. The first one, `create_ref` returns a `MyClass` with `ref` capability because `ref` is always default for class constructors. The second one `create_val` explicitly returns a `MyClass` with `val` capability.
```

"Capabilities" refers to the collection of refcaps in my mind, while "capability" refers to a single one of the refcaps -- is this a consistent thought, or do more people thing of refcaps in terms of their specific denials?

Reply:
I didn't make a conscious decision about it, and I assume that what you are expecting is consistent with the other documentation. In my mind, a reference capability type may have multiple capabilites in the sense that it may be capable of multiple things (specifically, "read", "write", and "share"). But much more likely that I just wasn't paying attention to the existing docs.